lib/matobjminimal_BROKEN.gi

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+##
+##  This file is part of GAP, a system for computational discrete algebra.
+##
+##  SPDX-License-Identifier: GPL-2.0-or-later
+##
+##  Copyright of GAP belongs to its developers, whose names are too numerous
+##  to list here. Please refer to the COPYRIGHT file for details.
+##
+
+############################################################################
+#
+# This file is a minimal implementation for new style vectors and matrices.
+# It stores matrices as a dense flat list.
+# This implementation implements only the methods required by the specification
+# for matrix objects. This is intended for testing.
+
+InstallMethod( NewMatrix,
+  "for IsMinimalBROKENMatrixRep, a ring, an int, and a list",
+  [ IsMinimalBROKENMatrixRep, IsRing, IsInt, IsList ],
+  function( filter, basedomain, nrcols, list_in )
+    local obj, filter2, list, rowindex, colindex;
+
+    # If applicable then replace a nested list 'list_in' by a flat list 'list'.
+    if Length(list_in) > 0 and (IsVectorObj(list_in[1]) or IsList(list_in[1])) then
+        list := [];
+        for rowindex in [1..Length(list_in)] do 
+            if Length(list_in[rowindex]) <> nrcols then
+                Error( "NewMatrix: Each row must have nrcols entries." );
+            fi;
+            for colindex in [1..nrcols] do 
+                list[(rowindex-1)*nrcols + colindex] := list_in[rowindex][colindex];
+            od;
+        od;
+    else
+        if Length(list_in) mod nrcols <> 0 then 
```

Comment:
You need to remove all trailing whitespace in this file, like on this line
```suggestion
        if Length(list_in) mod nrcols <> 0 then
```
Ideally, configure your editor to honor our `.editorconfig`, see https://editorconfig.org

Reply:
I still had the older .editorconfig where that was disabled in this branch. I
removed them now.